frontend_app_admin_workspace_user/src/component/AdminWorkspace.jsx

```diff
@@ -155,9 +156,8 @@ const AdminWorkspace = props => {
                     </td>
                     <td
                       className='table__sharedSpace adminWorkspace__workspaceTable__tr__td-link primaryColorFontHover'
-                      onClick={() => props.onClickWorkspace(ws.workspace_id)}
                     >
-                      {ws.label}
+                      <a href={PAGE.WORKSPACE.ADVANCED_DASHBOARD(ws.workspace_id)}>{ws.label}</a>
```

Comment:
You cannot use the `<a>` tag here because all redirections are made with ReactRouter.
You must use the `<Link>` component from react-router-dom here.

Also, I'd like you to add the classes `primaryColorFont` and `primaryColorFontDarkenHover` to style the text color please.
You'll can then also remove the `primaryColorFontHover` from the parent's `<td>`

Reply:
solved in [95b7c6a](https://github.com/tracim/tracim/pull/6149/commits/95b7c6a1d072be4bbd2675884c143f88cf7f3819)